respy/simulate.py

```diff
@@ -138,25 +138,39 @@ def _simulate_data(state_space, base_draws_sim, base_draws_wage, optim_paras, op
 
     base_draws_wage_transformed = np.exp(base_draws_wage * optim_paras["meas_error"])
 
+    # Create observables for simulation and store them in an extra container that
```

Comment:
Create a dataframe of appropriate length with one column period and all zeros here. Then, you do not need the if statement below.